src/main/resources/com/typesafe/sbt/packager/archetypes/java_server/systemloader/systemv/start-rpm-template

```diff
@@ -34,8 +34,6 @@
 # $JAVA_OPTS used in $RUN_CMD wrapper
 export JAVA_OPTS
 
-prog="${{exec}}"
```

Comment:
I thought it helped the readability of the file. Seeing as it’s already templated, it seems easier to see where the `app_name` variable flows to, rather than having the indirection of `prog`.